File: sdks/thunkmetrc-cs/ThunkMetrc.Wrapper/Models/CreateGenerateRequest.cs
using System;
using System.Collections.Generic;

namespace ThunkMetrc.Wrapper.Models
{
    public class CreateGenerateRequest
    {
        [global::System.Text.Json.Serialization.JsonPropertyName("PackageLabel")]
        public string? PackageLabel { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("Quantity")]
        public int? Quantity { get; set; }
    }
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/CreateGenerateRequest.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class CreateGenerateRequest(
    @JsonProperty("PackageLabel")
    val packageLabel: String? = null,
    @JsonProperty("Quantity")
    val quantity: Int? = null
)
